package.json

```diff
@@ -44,7 +44,7 @@
     "tslint-clean-code": "^0.2.6",
     "tslint-microsoft-contrib": "^5.0.3",
     "typescript": "^2.7.2",
-    "unibeautify": "^0.10.1"
+    "unibeautify": "0.11.1"
```

Comment:
Should this be a range?

Reply:
Yes it should, the caret must have been removed during the link.